README.md

```diff
@@ -6,7 +6,8 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 
 - `odf set recovery-profile <profile>`: Set the recovery profile to favor new IO, recovery, or balanced mode with options `high_client_ops`, `high_recovery_ops`, or `balanced`. The default is `balanced`.
 - `odf get recovery-profile`: Get the recovery profile value.
-- `odf set log-level <ceph-daemon> <log-level>`: Set the log level for Ceph daemons like OSD, mon, mds etc.
+- `odf set log-level <ceph-daemon> <ceph-subsystem> <log-level>`: Set the log level for Ceph daemons like OSD, mon, mds etc.
```

Comment:
We are going to need differentiation for setting log levels of different ODF components. Besides these ceph log levels, in the future we will want commands for the rook, noobaa, ocs-operator, or other component log levels. So I'm imagining the component as a prefix to log-level, though may need some discussion separate from this PR to finalize. And since `ceph-` is in the command name, no need for `ceph` in the param names.
```suggestion
- `odf set ceph-log-level <daemon> <subsystem> <log-level>`: Set the log level for Ceph daemons like OSD, mon, mds etc.
```

Reply:
That's a really good point. Adding a prefix before log level like you mentioned sounds good.

Reply:
Maybe that's where the concept of realms might have come in handy.  Something like: 
``` odf ceph set log-level <daemon> <subsystem> <log-level>
odf rook set log-level ....```

Reply:
now using `ceph-log-level`. Updated the README and doc file.

Reply:
Still needs discussion, I could go either way on these options, but let's keep the first option for now.
```
odf set ceph-log-level ....
odf ceph set log-level ...
```
But I don't want to call it a realm, since it's really only a component name.